whitesands/code/modules/metacoin/metacoin.dm

```diff
@@ -57,6 +58,7 @@ VOID CREW EDIT
 
 /client/proc/process_ten_minute_living()
 	inc_metabalance(METACOIN_TENMINUTELIVING_REWARD, FALSE)
+	give_random_ship()
```

Comment:
random ship part every 10 minutes? Hmm, that feels like a lot compared to metacoins

Reply:
It does, but its split into 5 groups. So if your aiming for certain ships it would be around 50 minutes per part of average, so for a 3 part thats 2 1/2 hours still way less then currently. But i don't think thats too too bad